src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.model.dto.request;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
+public class UserRequestDto {
+    @Email
```

Comment:
Are you really need to validate Dto in this HW? Why then you validate only this Dto?

Reply:
in order for the annotations to work, you need to write @Validation before the @REquestBody annotation in the controller